scripts/makefile/info.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
```

Comment:
Should this be called chart-info.sh instead?

Reply:
Maybe service info? It prints more than just helm chart info